cluster/gce/config-default.sh

```diff
@@ -30,6 +30,7 @@ REGIONAL_KUBE_ADDONS=${REGIONAL_KUBE_ADDONS:-true}
 NODE_SIZE=${NODE_SIZE:-n1-standard-2}
 NUM_NODES=${NUM_NODES:-3}
 MASTER_SIZE=${MASTER_SIZE:-n1-standard-$(get-master-size)}
+MASTER_MIN_CPU_ARCHITECTURE=${MASTER_MIN_CPU_ARCHITECTURE:-} # To allow choosing better architectures. Also helps w/ issue #55777.
```

Comment:
I vote for removing the "Also helps w/ issue #55777." part of the comment.
Same below.

Reply:
Yeah.. that sounds reasonable ;)
Done.